src/main/java/de/tum/in/www1/artemis/web/rest/dto/ResultDTO.java

```diff
@@ -21,16 +22,27 @@ public record ResultDTO(Long id, ZonedDateTime completionDate, Boolean successfu
         Integer codeIssueCount) {
 
     @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    public record FeedbackDTO(String text, String detailText, boolean hasLongFeedbackText, String reference, Double credits, Boolean positive, FeedbackType type,
-            Visibility visibility) {
+    public record FeedbackDTO(long id, String text, String detailText, boolean hasLongFeedbackText, String reference, Double credits, Boolean positive, FeedbackType type,
+            Visibility visibility, TestCaseDTO testCase) {
 
         public static FeedbackDTO of(Feedback feedback) {
-            return new FeedbackDTO(feedback.getText(), feedback.getDetailText(), feedback.getHasLongFeedbackText(), feedback.getReference(), feedback.getCredits(),
-                    feedback.isPositive(), feedback.getType(), feedback.getVisibility());
+            return new FeedbackDTO(feedback.getId(), feedback.getText(), feedback.getDetailText(), feedback.getHasLongFeedbackText(), feedback.getReference(),
+                    feedback.getCredits(), feedback.isPositive(), feedback.getType(), feedback.getVisibility(), TestCaseDTO.of(feedback.getTestCase()));
 
         }
     }
 
+    @JsonInclude(JsonInclude.Include.NON_EMPTY)
+    public record TestCaseDTO(String testName, Long id) {
```

Comment:
to be consistent with the dto above, some javadoc would be nice.

Reply:
I don't have any javadoc for any of these inner classes and I also don't see any useful information to be included here.